dotCMS/src/curl-test/ContentTypeResourceTests.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
please add a description for each test case to know what scenarios are being testing in case of failure

Reply:
Description added in json description fields.